docs/modules/prompts/getting_started.md

```diff
@@ -94,6 +94,18 @@ prompt.format(history="", input="What is 1 + 1?")
 
 You can read more about LangChainHub and the prompts available with it [here](https://github.com/hwchase17/langchain-hub).
 
+## Load a prompt template from the Hugging Face Hub
+
+The Hugging Face Hub [LangChainHub](https://huggingface.co/LangChainHub) organization contains a collection of prompts which can be loaded directly via LangChain.
```

Comment:
I've found [LangChainHub-Agents](https://huggingface.co/LangChainHub-Agents), [LangChainHub-Chains](https://huggingface.co/LangChainHub-Chains) and [LangChainHub-Prompts](https://huggingface.co/LangChainHub-Prompts) but no [LangChainHub](https://huggingface.co/LangChainHub). What is the plan for those orgs?

Reply:
We still need to update the docs. Originally we had a single org, now we have 3 orgs. I need to update the docs